src/config/rt.rs

```diff
@@ -49,13 +64,30 @@ impl RtcBuild {
         let final_dist = final_dist.canonicalize().context("error taking canonical path to dist dir")?;
         let staging_dist = final_dist.join(super::STAGE_DIR);
 
+        // Highlander-rule: There can be only one (prohibits contradicting arguments):
+        let cargo_features = if (opts.all_features && opts.no_default_features)
```

Comment:
Do you see a better way of checking for multiple `true`s; or is that even necessary? Should we allow passing more than one feature-arguments to cargo?